css/css-animations/parsing/keyframes-name-invalid.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>CSS Animations: parsing @keyframes name with invalid values</title>
+    <link rel="author" title="yisibl(一丝)" href="https://github.com/yisibl"/>
+    <link rel="help" href="https://drafts.csswg.org/css-animations/#typedef-keyframes-name">
+    <meta name="assert" content="@keyframes name supports the full grammar '<custom-ident> | <string>'.">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/css/support/parsing-testcommon.js"></script>
+</head>
+<body>
+    <div>
+        <main id="main"></main>
+    </div>
+    <script>
+        function cleanup_main() {
```

Comment:
maybe put these four functions in a helper file rather than having them duplicated across two tests?  Or just combine the two tests into one file?